research/flamby/fed_isic2019/ditto_mkmmd/README.md

```diff
@@ -0,0 +1,24 @@
+### Running hyperparameter sweep
+
+To run the hyperparameter sweep you simply run the command
+
+```bash
+./research/flamby/fed_isic2019/ditto_mkmmd/run_hp_sweep.sh \
+   path_to_config.yaml \
+   path_to_folder_for_artifacts/ \
+   path_to_folder_for_dataset/ \
+   path_to_desired_venv/
+```
+
+from the top level directory of the repository
+
+An example is something like
+``` bash
+./research/flamby/fed_isic2019/ditto_mkmmd/run_hp_sweep.sh \
+   research/flamby/fed_isic2019/ditto_mkmmd/config.yaml \
+   research/flamby/fed_isic2019/ditto/ \
```

Comment:
I think you want /ditto_mkmmd/ for this line as well?

Reply:
That is basically how I save my artifacts under ditto model but I can update it to ditto_mkmmd.